From d3fb1c32746580bb109ac21ff349a655a01c46f0 Mon Sep 17 00:00:00 2001
From: Pierre-Yves Chibon <pingou@pingoured.fr>
Date: Wed, 12 Aug 2020 20:47:22 +0200
Subject: [PATCH 2/9] Show the assignee's avatar on the board

If the ticket shown in the board is assigned to someone, show this
person's avatar. This helps not only seeing what is in progress or
blocked but also who is working on what or being blocked.

Fixes https://pagure.io/pagure/issue/4959

Signed-off-by: Pierre-Yves Chibon <pingou@pingoured.fr>
---
 pagure/templates/board.html | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pagure/templates/board.html b/pagure/templates/board.html
index 26509ae0..46a7eb65 100644
--- a/pagure/templates/board.html
+++ b/pagure/templates/board.html
@@ -65,6 +65,9 @@
             {% elif bissue.issue.status == 'Closed' %}
               <span class="fa fa-fw text-danger fa-exclamation-circle pt-1 icon_id"></span>
               <span class="text-danger font-weight-bold id_txt">#{{ bissue.issue.id }}</span>
+            {% endif %}
+            {% if bissue.issue.assignee %}
+            - {{ bissue.issue.assignee.username | avatar(size=20) | safe}}
             {% endif %}
               - {{ bissue.issue.title | truncate(80, False, '...') }}
               </a>
-- 
2.26.2

